mlflow/keras.py

```diff
@@ -400,6 +400,11 @@ def on_train_begin(self, logs=None):  # pylint: disable=unused-argument
             summary = '\n'.join(sum_list)
             try_mlflow_log(mlflow.set_tag, 'summary', summary)
 
+            with open('summary.txt', 'w') as f:
```

Comment:
Maybe write to a temporary file created via [`tempfile.mkstemp()`](https://docs.python.org/3/library/tempfile.html#tempfile.mkstemp), just to be safe/guarantee that we don't overwrite files in the user's working directory

Reply:
Good catch, will do